gitleaves/reports.py

```diff
@@ -86,12 +103,17 @@ def gen_ghwiki_reports():
     next_leaves_by_month = ((calendar.month_name[month], leaves)
                             for month, leaves
                             in next_leaves_by_month_temp_jar.items())
+    export_month_csv(data)
```

Comment:
For now: Drop this function `export_month_csv` just move the function code here

Reply:
Done